watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -460,6 +477,50 @@ def initialize(
 
         # Fix state variables
         flags = fix_state_vars(self, state_args)
+
+        # initialize vars caculated from state vars
+        for k in self.keys():
+            for j in self[k].params.component_list:
+                if self[k].is_property_constructed("mass_frac_phase_comp"):
+                    self[k].conc_mass_phase_comp["Liq", j].set_value(
+                        self[k].dens_mass_phase["Liq"]
+                        * self[k].mass_frac_phase_comp["Liq", j]
+                    )
```

Comment:
This section doesn't seem right to me. You check to see if `mass_frac` is constructed, but then you are initializing `conc_mass` instead. Is something missing here? Shouldn't this line check to see if `conc_mass` is constructed then initializing it? 

Shouldn't there also be a step before where you initialize 'mass_frac'? I would think that the initialization of the mole fractions should be the first step (since it only depends on the state var `flow_mol`). Then you can initialize the `mass_frac` based on the `mole_frac` (since they are closely related).

Reply:
Per conversion with Xiangyu, this section is correct. The existence of mass_frac already implies that conc_mass is constructed.

Reply:
Thanks for @aladshaw3 comment.  Actually I did find an error in a second look and fixed it in the latest commit.   Also, I went ahead to add initialization methods for a few other vars.